apis/contentapi/contentapi.yaml

```diff
@@ -1822,7 +1822,7 @@ paths:
   /content/v1/search:
     post:
       description: |-
-        This API is associated with Searching Object created as part of sourcing object creation on the Sunbird Platform.
+        This API is associated with Searching content and collection objects created as part of sourcing object creation on the Sunbird Platform.
```

Comment:
1. Line 314, 315, 438, &439: Please rectify the formatting of the bullet points. It is appearing as sub-bullets.
2. Global: Please check the punctuation in the line “The Content Type entity is the media type of the resource.Possible media types can be”
3. Global: Please rectify the casing of “To” in the line “Please append a valid Content Id To the Request URL”
4. Line 441: Please recheck the sentence formation
5. Line 1362, 1684, & 2224: Please insert a space before endpoint in “**endpoint executes request”
6. Line 1496: remove single tick at the end of the line
7. Line 398: Rectify the punctuations in line 398
8. Line 2322, 2698, 2996: Please change the casing of Url to URL
9. Line 1026, 1531: Please rectify the formatting in the line success response.
10. Global: Please rectify the punctuations and spacings in all the response descriptions.

Reply:
@anisha147 Not feasible to address punctuations and spacings in all the response descriptions now. We can look into this gradually. However, I have tried to address issues at places similar to the line numbers you have mentioned.